FreeRTOS/FreeRTOS-Plus-TCP
# Update minor fixes in Readme

Description
-----------

- Update README.md to move V3.1.0 release update.

Test Steps
-----------
NA

Checklist:
----------


- [ NA ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
